### PR TITLE
emit metrics for pod killing and add tags for victims

### PR DESCRIPTION
**Pull request by waynepeking348**
#### What type of PR is this?
Enhancements

#### What this PR does / why we need it:
during evicting process, we miss tags for pod info, add this pr to make up this info

### CODE REVIEW

**Review by luomingmeng — APPROVED** [review]

File: pkg/agent/evictionmanager/podkiller/killer.go [hunk 1]
```diff
@@ -201,21 +208,28 @@ func deleteWithRetry(pod *v1.Pod, gracePeriod int64, timeoutDuration time.Durati
...
 	timeoutDuration := getWaitingPeriod(gracePeriodSeconds)
 	klog.Infof("[killer] evict pod %v/%v with graceful seconds %v", pod.Namespace, pod.Name, gracePeriodSeconds)
 
 	if err := deleteWithRetry(pod, gracePeriodSeconds, timeoutDuration, evictPod); err != nil {
 		recorder.Eventf(pod, nil, v1.EventTypeWarning, consts.EventReasonEvictFailed, consts.EventActionEvicting,
 			fmt.Sprintf("Evict failed: %s", err))
+		_ = emitter.StoreInt64(MetricsNameKillPod, 1, metrics.MetricTypeNameRaw,
+			metrics.MetricTag{Key: "state", Val: "failed"},
+			metrics.MetricTag{Key: "pod_ns", Val: pod.Namespace},
+			metrics.MetricTag{Key: "pod_nme", Val: pod.Name})
```
Comment (caohe) [inline pkg/agent/evictionmanager/podkiller/killer.go:222]: typo: pod_name

**Review by caohe — CHANGES_REQUESTED** [review]

File: pkg/agent/evictionmanager/podkiller/killer.go [hunk 2]
```diff
@@ -201,21 +208,28 @@ func deleteWithRetry(pod *v1.Pod, gracePeriod int64, timeoutDuration time.Durati
...
 
 		return fmt.Errorf("evict failed %v", err)
 	}
 
 	recorder.Eventf(pod, nil, v1.EventTypeNormal, consts.EventReasonEvictCreated, consts.EventActionEvicting,
 		"Successfully create eviction; reason: %s", reason)
+	_ = emitter.StoreInt64(MetricsNameKillPod, 1, metrics.MetricTypeNameRaw,
+		metrics.MetricTag{Key: "state", Val: "succeeded"},
+		metrics.MetricTag{Key: "pod_ns", Val: pod.Namespace},
+		metrics.MetricTag{Key: "pod_nme", Val: pod.Name})
```
Comment (caohe) [inline pkg/agent/evictionmanager/podkiller/killer.go:232]: typo

File: pkg/agent/evictionmanager/podkiller/killer.go (hunk 1 above)
Comment (waynepeking348) [reply pkg/agent/evictionmanager/podkiller/killer.go]: fixed

File: pkg/agent/evictionmanager/podkiller/killer.go (hunk 2 above)
Comment (waynepeking348) [reply pkg/agent/evictionmanager/podkiller/killer.go]: fixed

**Review by caohe — APPROVED** [review]
LGTM

**Review by csfldf — APPROVED** [review]

**Review by luomingmeng — APPROVED** [review]

**Pull request merged**